content/docs/handling-events.md

```diff
@@ -91,64 +91,64 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
+[**Essayez-le dans CodePen**](http://codepen.io/gaearon/pen/xEmzGg?editors=0010)
 
-You have to be careful about the meaning of `this` in JSX callbacks. In JavaScript, class methods are not [bound](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) by default. If you forget to bind `this.handleClick` and pass it to `onClick`, `this` will be `undefined` when the function is actually called.
+En JSX, vous devez être prudent avec l'utilisation de `this` dans les fonctions de rappels. En JavaScript, les méthodes de classes ne sont pas [liées](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) par défaut (Dans la suite de l'article, pour des raisons de concision, nous utiliserons le terme générique anglais bind, NdT). Si vous oubliez de bind `this.handleClick` et si vous l'utilisez dans `onClick`, `this` sera `undefined` quand la fonction sera appelée.
 
-This is not React-specific behavior; it is a part of [how functions work in JavaScript](https://www.smashingmagazine.com/2014/01/understanding-javascript-function-prototype-bind/). Generally, if you refer to a method without `()` after it, such as `onClick={this.handleClick}`, you should bind that method.
+Ce n'est pas un comportement spécifique à React, cela fait en effet partie du [fonctionnement général des fonctions en JavaScript](https://www.smashingmagazine.com/2014/01/understanding-javascript-function-prototype-bind/). En général, si vous faites références à une méthode sans `()`, par exemple `onClick={this.handleClick}`, vous devriez utiliser une liaison de données.
 
-If calling `bind` annoys you, there are two ways you can get around this. If you are using the experimental [public class fields syntax](https://babeljs.io/docs/plugins/transform-class-properties/), you can use class fields to correctly bind callbacks:
+Si vous ne souhaitez pas utiliser `bind`, vous avez alors deux solutions. Si vous avez l'habitude d'utiliser la [syntaxe des champs de classes](https://babeljs.io/docs/plugins/transform-class-properties/), qui sont à l'état expérimental, vous pourriez alors les utiliser pour faire la liaison avec les fonctions de rappels.
 
 ```js{2-6}
 class LoggingButton extends React.Component {
-  // This syntax ensures `this` is bound within handleClick.
-  // Warning: this is *experimental* syntax.
+  // Cette syntaxe nous assure que `this` est bien lié avec la méthode handleClick.
+  // Attention: cette syntaxe est encore à l'état *expérimental*.
   handleClick = () => {
-    console.log('this is:', this);
+    console.log('Voici:', this);
```

Comment:
```suggestion
    console.log('this vaut :', this);
```

Ah ah, c'était pas dans le sens naturel, mais “this” au sens technique. Piège !

Reply:
Oui et j'avais pas compris du tout ! :D